cmd/intervalSubsetMatrix/intervalSubsetMatrix.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/interval"
+	"log"
+	"sort"
+)
+
+func intervalSubsetMatrix(unionFile string, fileListFile string, outFile string) {
+	var err error
+	var i int
+	var j interval.Interval
+	var currOverlaps []interval.Interval
+	var currIntervalChan <-chan interval.Interval
+	var unionIntervals = make([]interval.Interval, 0)
+	var currLineString string
+
+	unionRecChan := interval.GoReadToChan(unionFile)
+	files := fileio.Read(fileListFile)
+	out := fileio.EasyCreate(outFile)
+
+	for curr := range unionRecChan {
```

Comment:
Should we make a simple "Read" function for intervals at some point in the future?